packages/react-data-grid/src/Canvas.tsx

```diff
@@ -241,7 +272,8 @@ export default function Canvas<R>({
       colOverscanEndIdx,
       lastFrozenColumnIndex: columnMetrics.lastFrozenColumnIndex,
       isScrolling,
-      scrollLeft
+      scrollLeft,
+      isSummaryRow: summaryRows && summaryRows.includes(row)
```

Comment:
This will always be false, correct?

Reply:
It will always be `false` only when `summaryRows` doesn't exist. Same story like `isRowSelected`, if it's not enabled, it will just be false and populated to next levels.